Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: software/install-cli.md

---
title: "Install the Astro CLI"
sidebar_label: "Install the CLI"
id: install-cli
description: Establish a local testing environment and deploy to Astronomer Software from the CLI.
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
import {siteVariables} from '@site/src/versions';

## Overview

This guide provides instructions for how to install the Astro CLI.

> The Astro CLI is the easiest way to run Apache Airflow on your machine. From the CLI, you can run a local Apache Airflow environment with a dedicated Webserver, Scheduler and Postgres Database. Once you create an Astronomer Software project, you can easily customize it (e.g. add Python or OS-level packages, plugins etc.) and test it on your local machine.

Reviewer: ```suggestion
The Astro CLI is the easiest way to run Apache Airflow on your machine. From the Astro CLI, you can run a local Apache Airflow environment with a dedicated Webserver, Scheduler and Postgres Database. Once you create an Astronomer Software project, you can customize it (for example, add Python or OS-level packages or add plugins) and test it on your local machine.
```

You can also use the CLI to:

- Authenticate to Astronomer Software.

> - List the Astro Workspace and Deployments you have access to.

Reviewer: ```suggestion
- List the Astro Workspace and Deployments you can access.
```
